typescript/packages/well-log-viewer/src/SyncLogViewer.tsx

```diff
@@ -416,9 +432,13 @@ class SyncLogViewer extends Component<SyncLogViewerProps, State> {
         if (
             this.props.syncContentSelection !==
                 prevProps.syncContentSelection ||
-            this.props.welllogs.length !== prevProps.welllogs.length
+            this.props.welllogs?.length !== prevProps.welllogs?.length
```

Comment:
this.props.wellLogCollections?.length ?? this.props.welllogs?.length !== prevProps.wellLogCollections?.length ?? prevProps.welllogs?.length